src/client/opamConfigCommand.ml

```diff
@@ -673,6 +673,9 @@ let global_allowed_fields, global_allowed_sections =
         "jobs", Atomic,
         Config.with_jobs_opt
           (InitConfig.jobs in_config ++ Config.jobs Config.empty);
+        "archive-mirrors", Atomic,
```

Comment:
This field is a list of urls, it should be `Modifiable` (appending urls) not `Atomic` (replacement)